modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing the {pipelines-title} Operator.
+*** To install a specific version of the {pipelines-title} Operator, cluster adminsitrators can also the `pipelines-<version>` channel. For example, to install the {pipelines-title} Operator version `1.8.x`, you can use the `pipelines-1.8` channel.
```

Comment:
1. typo adminsitrators
2. can also ... some verb missing ... the `pipelines-<version>` channel